src/OSPSuite.Presentation/Presenters/Importer/ColumnMappingPresenter.cs

```diff
@@ -371,7 +371,7 @@ public IEnumerable<RowOptionDTO> GetAvailableRowsFor(ColumnMappingDTO model)
          if (model.CurrentColumnType == ColumnMappingDTO.ColumnType.MetaData)
          {
             var metaDataCategory = _metaDataCategories.FirstOrDefault(md => md.Name == model.MappingName);
-            if (model.Source != null && !(model.Source as MetaDataFormatParameter).IsColumn && !metaDataCategory.ListOfValues.Keys.Union(excelColumns).Contains(model.ExcelColumn))
+            if (model.Source != null && (model.Source as MetaDataFormatParameter).ColumnName != null && !(model.Source as MetaDataFormatParameter).IsColumn && !metaDataCategory.ListOfValues.Keys.Union(excelColumns).Contains(model.ExcelColumn))
```

Comment:
Man this is SO complicated here.I'd suggest to create a private property that returns this expression
And in the property, explain what are all those checks about
Don't hesitate to simplify the coedw


if (model.source==null) 
return false

etc..

in fact the firs tline cound be var formatParameter = model.Source as MetaDataFormatParameter;
if(formatParameter==null)
return false

then your code will be much easier to understand. I have personally no idea what's going on